plugins/setup/src/main/java/com/wazuh/setup/index/WazuhIndices.java

```diff
@@ -135,7 +145,12 @@ public void initialize() {
         this.indexTemplates.forEach(
                 (k, v) -> {
                     this.putTemplate(k);
-                    this.putIndex(v);
+                    if (Objects.equals(v, "wazuh-*-users")) {
+                        this.putIndex(".internal-users");
+                        this.putIndex("wazuh-custom-users");
+                    } else {
+                        this.putIndex(v);
+                    }
```

Comment:
Maybe we can avoid this as follows:

```java
this.indexTemplates.put("index-template-users", ".internal-users");
this.indexTemplates.put("index-template-users", "wazuh-custom-users");
```

We need to review the index names.

Reply:
That is not possible since the mapping does not let us use duplicated keys, but I've found another way to handle this.